discovery-provider/alembic/versions/5bcbe23f6c70_user_track_collection_mat_views.py

```diff
@@ -0,0 +1,238 @@
+"""user-track-collection-mat-views
+
+Revision ID: 5bcbe23f6c70
+Revises: 2ff46a8686fa
+Create Date: 2021-04-12 20:01:40.395480
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5bcbe23f6c70'
+down_revision = '2ff46a8686fa'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    connection = op.get_bind()
+    connection.execute('''
+      begin;
+        --- ======================= AGGREGATE USER =======================
+        DROP MATERIALIZED VIEW IF EXISTS aggregate_user;
```

Comment:
why drop beforehand? i dont see any problems w/ it, just curious

Reply:
I copied this query back and forth from a sql query tool - no practical reason why it's here.